addons/mail/static/tests/discuss/discuss_tests.js

```diff
@@ -177,4 +178,30 @@ QUnit.module("mail", (hooks) => {
             target.querySelector(".o-mail-composer-textarea")
         );
     });
+
+    QUnit.test("posting a message right after another one is squashed", async (assert) => {
+        const def = new Deferred();
+        let flag = false;
+        const server = new TestServer();
+        server.addChannel(1, "general", "General announcements...");
+        const env = makeTestEnv(async (route, params) => {
+            if (flag && route === "/mail/message/post") {
+                await def;
```

Comment:
unless the `def` is expected to be resolved at some point, couldn't this just be an empty promise?